osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -80,8 +85,10 @@ public DrawableSpinner([CanBeNull] Spinner s = null)
         {
         }
 
+#nullable enable
```

Comment:
we don't do this anywhere else, please remove, what you want is probably `[BackgroundDependencyLoader(true)]` on the BDL method to stop it failing